ClientRequirments.md

```diff
@@ -0,0 +1,37 @@
+# Platform Requirements for Fluid Framework Client Packages
+
+These are the platform requirements for the current version of Fluid Framework Client Packages.
+These are intentionally quite strict as they have to be supported for the entire LTS timeline for LTS versions.
+These restrictions can be loosened over time, adding support for specific configurations desired by users of the client packages.
+
+It is likely the other configurations will work, but there are not "supported" and thus if they stop working it is not considered a breaking change.
```

Comment:
It is likely *that* other configurations will work, but *as they* are not supported, if they stop working, that is not a breaking change.